### ISSUE TITLE
Add RNN support to torch/PPO and torch/TRPO

### ISSUE THREAD

**Issue by ryanjulian** [rlworkgroup/garage#1181]

**Comment by irisliucy** [comment]
I'm planning for the following implementations for adding RNN support to PPO/ TRPO:
1. RNN:  `lstm_module`
2. RNN policies and its module for PPO: `guassianLSTMModule`, `guassianLSTMPolicy`
3. examples (maybe Pendulum): `trpo_lstm`, `ppo_lstm`

Let me know if I miss anything!

**Comment by ryanjulian** [comment]
Yes, that sounds good! You should probably pick the more popular of LSTM and GRU.

**Comment by nikhilxb** [comment]
> I'm planning to implement the followings to add RNN support to PPO/ TRPO:
> 
> * [x]  RNN:  `lstm_module`
> * [x]  RNN policies and its module for PPO: `guassianLSTMModule`, `guassianLSTMPolicy`
> * [x]  examples (maybe Pendulum): `trpo_lstm`, `ppo_lstm`
> * [ ]  test cases for 1,2
> 
> Let me know if I miss anything!

@irisliucy Was this ever completed? I can't seem to find the code for it. I'm trying to implement a recurrent policy in PyTorch to use with PPO and can't figure out the API, so would be helpful to see these examples. Thanks!
